node/src/Pos/Launcher/Resource.hs

```diff
@@ -143,7 +143,21 @@ allocateNodeResources transport networkConfig np@NodeParams {..} sscnp = do
             futureLrcContext
     runInitMode initModeContext $ do
         initNodeDBs @ssc
-        ctx@NodeContext {..} <- allocateNodeContext np sscnp putSlotting networkConfig
+
+        -- EKG monitoring stuff.
+        --
+        -- Relevant even if monitoring is turned off (no port given). The
+        -- gauge and distribution can be sampled by the server dispatcher
+        -- and used to inform a policy for delaying the next receive event.
+        --
+        -- TODO implement this. Requires time-warp-nt commit
+        --   275c16b38a715264b0b12f32c2f22ab478db29e9
+        -- in addition to the non-master
+        --   fdef06b1ace22e9d91c5a81f7902eb5d4b6eb44f
+        -- for flexible EKG setup.
```

Comment:
This comment is almost certainly out of date. Can you check.

Reply:
Unfortunately I don't understand this comment. `275c16b38a715264b0b12f32c2f22ab478db29e9` doesn't exist in `time-warp-nt`.
I would like to leave it to @avieth (I guess he wrote this comment).
For now I just moved it slightly, because it might contain something useful.

Reply:
I think it's just stale. All the EKG stuff in time-warp has been merged. You can remove the comment.